src/ui/common/src/components/tables/KeyValueTable.tsx

```diff
@@ -0,0 +1,101 @@
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell, { TableCellProps } from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import * as React from 'react';
+
+import { Data, DataSchema } from '../../utils/data';
+import { CheckTableItem } from './CheckTableItem';
+
+export enum KeyValueTableType {
+  Metric = 'metric',
+  Check = 'check',
+}
+
+interface KeyValueTableProps {
+  rows: Data;
+  schema?: DataSchema;
+  height?: string;
+  width?: string;
+  maxHeight?: string;
+  stickyHeader?: boolean;
+  tableAlign?: string;
+  tableType: KeyValueTableType;
+}
+
+const kvSchema: DataSchema = {
+  fields: [
+    { name: 'Title', type: 'varchar' },
+    { name: 'Value', type: 'varchar' },
+  ],
+  pandas_version: '0.0.1', // TODO: Figure out what to set this value to.
```

Comment:
I'm not entirely sure what the pandas_version does with regards to how we render our table. If someone here can shed some light on this, that would be awesome. otherwise, we'll continue setting yet ignoring this value for now 🙂

Reply:
@cw75 @kenxu95?